skills/migrate-to-sizing/SKILL.md

```
@@ -0,0 +1,214 @@
+sizing_loader:
+  vcpu: 16             # override default (4)
+  memory: '>=16'       # override default (>=8)
+
+sizing_monitor:
+  vcpu: 4              # override default (2)
+  memory: '>=16'       # override default (>=8)
+```
+
+### Decision Tree: When to Pin Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)